services/items.go

```diff
@@ -0,0 +1,58 @@
+package services
```

Comment:
It's not necessary for now, but since for the next deliverables you will need to call an API, and the logic to do so could be considered as a service, you could probably make a dedicated csvservice and apiservice packages(these are not name suggestions, only the idea to separate them into their own files). This way the idea of a service is not tightly coupled to only csv files